packages/arb-token-bridge-ui/src/hooks/__tests__/useUpdateUsdcBalances.test.ts

```diff
@@ -0,0 +1,258 @@
+      {
+        sourceChain: getWagmiChain(ChainId.Base),
+        sourceChainProvider: getProviderForChainId(ChainId.Base),
+        destinationChain: getWagmiChain(geistMainnetChainId),
+        destinationChainProvider: getProviderForChainId(
+          geistMainnetChainId as ChainId
+        )
+      },
+      jest.fn()
+    ])
+
+    const { result } = renderHook(useParentChainUsdcAddress)
+    expect(result.current).toEqual(undefined)
+  })
+
+  it('should return undefined when source chain is an L3 Orbit chain on top of Base, e.g. Geist Mainnet, and destination chain is Base', () => {
+    mockedUseNetworks.mockReturnValue([
+      {
+        sourceChain: getWagmiChain(geistMainnetChainId),
+        sourceChainProvider: getProviderForChainId(
+          geistMainnetChainId as ChainId
+        ),
+        destinationChain: getWagmiChain(ChainId.Base),
+        destinationChainProvider: getProviderForChainId(ChainId.Base)
+      },
+      jest.fn()
+    ])
+
+    const { result } = renderHook(useParentChainUsdcAddress)
+    expect(result.current).toEqual(undefined)
+  })
+
+  it('should return undefined when source chain is Base Sepolia and destination chain is an L3 Orbit chain on top of Base Sepolia, e.g. Polter Testnet', () => {
+    mockedUseNetworks.mockReturnValue([
+      {
+        sourceChain: getWagmiChain(ChainId.BaseSepolia),
+        sourceChainProvider: getProviderForChainId(ChainId.BaseSepolia),
+        destinationChain: getWagmiChain(polterTestnetChainId),
+        destinationChainProvider: getProviderForChainId(
+          polterTestnetChainId as ChainId
+        )
+      },
+      jest.fn()
+    ])
+
+    const { result } = renderHook(useParentChainUsdcAddress)
+    expect(result.current).toEqual(undefined)
+  })
+
+  it('should return undefined when source chain is an L3 Orbit chain on top of Base Sepolia, e.g. Polter Testnet, and destination chain is Base Sepolia', () => {
+    mockedUseNetworks.mockReturnValue([
+      {
+        sourceChain: getWagmiChain(polterTestnetChainId),
+        sourceChainProvider: getProviderForChainId(
+          polterTestnetChainId as ChainId
+        ),
+        destinationChain: getWagmiChain(ChainId.BaseSepolia),
+        destinationChainProvider: getProviderForChainId(ChainId.BaseSepolia)
+      },
+      jest.fn()
+    ])
+
+    const { result } = renderHook(useParentChainUsdcAddress)
+    expect(result.current).toEqual(undefined)
+  })
+})
+
+describe('childChainUsdcAddressFetcher', () => {
+  it('should return native USDC address on Arbitrum One when parent USDC address is native USDC on Ethereum, parent chain is Ethereum, and child chain is Arbitrum One', async () => {
+    const result = await childChainUsdcAddressFetcher([
+      CommonAddress.Ethereum.USDC,
+      ChainId.Ethereum,
+      ChainId.ArbitrumOne
+    ])
+
+    expect(result).toEqual(CommonAddress.ArbitrumOne.USDC)
+    expect(result).not.toEqual(CommonAddress.ArbitrumOne['USDC.e'])
```

Comment:
nit: we don't need to check for USDC.e, if USDC passed then USDC.e must fail (unless we change `ArbitrumOne['USDC.e']` to the same value)